dcsazure_Snowflake_Legacy_to_Snowflake_Legacy_mask_pl/dcsazure_Snowflake_Legacy_to_Snowflake_Legacy_mask_pl.json

```diff
@@ -595,51 +599,92 @@
                 "referenceName": "[parameters('Metadata Datastore')]",
                 "type": "LinkedServiceReference"
               },
-              "name": "Ruleset"
+              "name": "Ruleset",
+              "description": "Get the ruleset table from the metadata store at DF_METADATA_SCHEMA.DF_METADATA_RULESET_TABLE"
             },
             {
               "linkedService": {
                 "referenceName": "[parameters('Metadata Datastore')]",
                 "type": "LinkedServiceReference"
               },
-              "name": "TypeMapping"
+              "name": "TypeMapping",
+              "description": "Get the type mapping table from the metadata store at DF_METADATA_STORE.DF_METADATA_ADF_TYPE_MAPPING_TABLE"
             }
           ],
           "sinks": [
             {
               "name": "MaskingParameterOutput",
-              "description": "Export data to cache"
+              "description": "Sink results of computing masking parameters to activity output cache"
             }
           ],
           "transformations": [
             {
-              "name": "FilterToSingleTable"
+              "name": "FilterToSingleTable",
+              "description": "Filter ruleset table down to the table in question by specifying dataset, specified_database, specified_schema, identified_table, and assigned_algorithm - making sure they match the dataset associated with each version of the dataflow, DF_SOURCE_DATABASE, DF_SOURCE_SCHEMA, DF_SOURCE_TABLE, and not empty or null (respectively). This filters the ruleset down to only the rules that need to be applied for masking this particular table"
+            },
+            {
+              "name": "RulesetWithTypes",
+              "description": "Join the ruleset table with the type mapping table based on the type of the column and the translation of that type to an ADF type"
+            },
+            {
+              "name": "RulesetWithAlgorithmTypeMapping",
+              "description": "Generate several columns:\n\n    output_row that always contains 1 (used later for Aggregate and Join operations)\n    adf_type_conversion that contains a string like <column_name> as <adf_type>\n    column_width_estimate that contains an integer that uses DF_COLUMN_WIDTH_ESTIMATE as the width for any column where identified_column_max_length is not positive, and identified_column_max_length plus some padding otherwise"
```

Comment:
Just curious, does the UI resolve the `\n`?

Reply:
It does :)